src/pages/FolderPage/FolderPage.tsx

```diff
@@ -0,0 +1,185 @@
+import { ChangeEvent, useState } from "react";
+import { NoResults } from "pages";
+import styles from "./folder.module.css";
+import Header from "./components/Header";
+import CardList from "components/CardList/CardList";
+import SearchBar from "components/Input/SearchBar/SearchBar";
+import { ReactComponent as AddIcon } from "assets/images/ic_add.svg";
+import DeleteModal from "components/Modal/DeleteModal/DeleteModal";
+import SharedModal from "components/Modal/SharedModal/SharedModal";
+import FolderModal from "components/Modal/FolderModal/FolderModal";
+import Category from "./components/Category/Category";
+import ActionButton from "./components/ActionButton/ActionButton";
+import useModal from "utils/hooks/useModal";
+import {
+  ALL,
+  DELETE_FOLDER,
+  ADD_FOLDER,
+  SHARED,
+  EDIT,
+} from "utils/constants/strings";
+import { GetLinkResponse, GetFolderResponse } from "types/apis";
+import { useGetFolders } from "utils/hooks/useGetFolders";
+import { useGetLinks } from "utils/hooks/useGetLinks";
+
+type Nullable<T> = T | null;
+
+export type SelectedCategory = {
+  id: Nullable<number>;
+  name: string;
+};
+
+export type ButtonClick = (
+  categoryId: Nullable<number>,
+  categoryName: string
+) => void;
+
+interface UseFetchResponse<T> {
+  data: Nullable<T>;
+  loading?: boolean;
+  error?: string;
+}
+
+function FolderPage() {
+  let userId = null;
+
+  const value = localStorage.getItem("userId");
+
+  if (typeof value === "string") {
+    userId = JSON.parse(value);
+  }
```

Comment:
이렇게 표현해줄 수 있을 것 같아요~
```suggestion
const userId = JSON.parse(localStorage.getItem('userId') ?? '')
```